### PR TITLE
Added documentation for known methods.

### PR DESCRIPTION
**Pull request by i3wgnit**
I have a few more prepared, however, more discussion is needed to confirm the
arguments.

### CODE REVIEW

**Comment by i3wgnit** [comment]
We should gitignore `docs/`
The deltas are stupid (+6k, -4k) for some documentation.

File: src/index.js [hunk 1]
```diff
@@ -143,6 +168,15 @@ export default class Client {
...
     return transaction
   }
+
+  /**
+   * Get a raw hexadecimal transaction given its hash.
+   * @param {!string} txHash - A hexadecimal string that represents the *hash* of the desired transaction. If `hash` is not a string, it will be converted through the `toString(16)` method.
+   * @return {Promise<string, null>} Returns the raw Transaction with the same hash as the given output. Returns null if no transaction was found.
+   */
+  async getRawTransactionByHash (txHash) {
+    this._checkMethod('getRawTransactionByHash')
```
Comment (harshjv) [inline src/index.js:176]: Did you forget to call `this.provider. getRawTransactionByHash`?

**Review by harshjv — CHANGES_REQUESTED** [review]
Can you fetch the latest changes and rebase? docs building on git commit is already disabled.

File: src/index.js (hunk 1 above)
Comment (i3wgnit) [reply src/index.js:176]: i didn't add any of the code in. Just the documentation. I could add it today if you want.

Comment (harshjv) [reply src/index.js:176]: okay then. make sure when you push commits, you rebase first :)

**Pull request closed without merging**